test/e2e/fixtures.go

```diff
@@ -323,6 +335,9 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 	// dump the logs for flow-aggregator Pods to disk.
 	data.forAllMatchingPodsInNamespace("", flowAggregatorNamespace, writePodLogs)
 
+	// dump the logs for flow-visibility Pods to disk.
+	data.forAllMatchingPodsInNamespace("", flowVisibilityNamespace, writePodLogs)
```

Comment:
should we get the logs from the CH operator Pod as well, just in case?

Reply:
added